src/components/RangeDatePicker.tsx

```diff
@@ -1,8 +1,13 @@
 import * as React from 'react';
 import DateTimePicker from 'react-native-modal-datetime-picker';
 import FontAwesome from 'react-native-vector-icons/FontAwesome';
-import { formatDate } from '../modules/format';
+import { formatDate, getLocale } from '../modules/format';
 import { getTheme } from '../modules/theme';
+import { DateRange, SelectedRanges } from 'react-date-range';
+import * as ReactDOM from 'react-dom';
```

Comment:
Não é bom importar o react-dom quando for nativo, a maioria dos projetos não vai ter ele como dependência.

Pensando melhor, vai ter muito "if web" aqui, o que pode tornar difícil importar apenas o que for usado por cada plataforma. Talvez esse seja o caso de fazer um `RangeDatePicker.web.tsx`: https://github.com/necolas/react-native-web/blob/master/docs/guides/multi-platform-apps.md#web-specific-code